otx/cli/tools/train.py

```diff
@@ -134,17 +153,40 @@ def main():
     dataset_class = get_dataset_class(template.task_type)
 
     # Create instances of Task, ConfigurableParameters and Dataset.
+    train_ann_files, train_data_roots = args.train_ann_files, args.train_data_roots
```

Comment:
Same here with eval and export. If a user wants to reuse training algorithm(template) with another dataset(not data.yaml)?

Reply:
Corrected and explained the same in the comments above.